bin/wiki_entity_linking/entity_linker_evaluation.py

```diff
@@ -0,0 +1,247 @@
+import logging
+import random
+
+from collections import defaultdict
+from tqdm import tqdm
+from typing import Dict, NamedTuple
+
+logger = logging.getLogger(__name__)
+
```

Comment:
Thanks for updating all the `print` statements - definitely more clean solution to work with a proper logger instead :-)